### DISCUSSION TITLE
RabbitMQ password with special characters not supported

### DISCUSSION THREAD

**Discussion by stereu** [postalserver/postal#2332] [Help with using Postal]
## Bug Description

Setting a RabbitMQ password with special characters like a "hashtag" during rabbitmq container creating and in `/opt/postal/config/postal.yml` leads to `postal-requeuer-1` and `postal-worker-1` crashing due to no connection to rabbitmq container.

## Errorlog (postal-requeuer)

`E, [2023-01-22T09:31:06.182392 #8] ERROR -- #<Bunny::Session:0x56061734c6f0 postal@127.0.0.1:5672, vhost=postal, addresses=[127.0.0.1:5672]>: Authentication with RabbitMQ failed: 403 ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.
rake aborted!
Bunny::AuthenticationFailureError: Authentication with RabbitMQ failed. Please check your connection settings. Username: postal, vhost: postal, password length: 11
`

**Comment by willpower232** [comment]
I think thats a YAML thing, you probably just need to wrap the value in `"`
